test/parallel/test-event-emitter-add-listeners.js

```diff
@@ -95,3 +95,14 @@ assert.throws(() => {
   message: 'The "listener" argument must be of type function. ' +
            'Received null'
 });
+
+// Verify that type must be string or symbol
+assert.throws(() => {
+  const ee = new EventEmitter();
+  ee.on(() => {}, () => {});
```

Comment:
What about checking for all prohibited values? You could use the `common.invalidArgTypeHelper()` to get the actual error message or just use a regular expression on the message.

Reply:
done